woocommerce/woocommerce-android
# [Payment Method Improvements] Cash Payment Improvements

Closes: #11413

### Description  

This PR introduces a new "Change Due" calculator screen for merchants processing cash payments, enhancing the existing cash handling functionalities in our mobile app. 

The new screen will only be visible when the 'OTHER_PAYMENT_METHODS' feature flag is enabled, ensuring that this functionality can be tested incrementally and does not impact users negatively during the roll-out phase.

### Testing instructions  

1. Turn on the 'OTHER_PAYMENT_METHODS' feature flag.
2. Navigate to the payment capture screen and select the cash payment option.
3. Verify that the "Change Due" calculator screen appears 
 
 The following will be in a subsequent PR for easier review:
   - The "Cash Received" field displays the total amount due by default.
   - The UI corresponds to the new design specifications and not the current iOS design.
* Edit the amount in the "Cash Received" field and ensure the displayed change due updates accordingly.
*  Tap "Mark order as complete", and confirm that it functions similarly to the "Mark as paid" button in the current dialog.
*  The new calculator will allow merchants to easily calculate the change owed to a customer when capturing a cash payment. 
* Complete unit test(s)

### Images/gif  

_Images or gifs demonstrating the new screen and comparisons to the previous version will be attached here._
